test/test.js

```diff
@@ -14,79 +14,116 @@ describe('BirthdayReminder', () => {
         });
     });
     describe('#run()', () => {
-        it('sends a signal with the name of the birthday person in the signal message when it is the birthday date + check if the keyboard key', async function () {
-            const config = getConfigForDate(new Date());// create a configuration: the birthday date is the current date
-            console.log('<<<<<<', config.applet.user.monthOfTheBirthday+5);
+        it('checks the birthday name & if selected keyboard key blinks', async function () {
+            // create a configuration: the birthday date is the current date
+            const config = getConfigForDate(new Date());
             let app = await buildApp(config);
             return app.run().then((signal) => {
-                assert.ok(signal.points);
-                assert(signal.message.includes(config.applet.user.nameOfTheBirthdayPerson));// we check if we receive the signal with the birthday name
+                assert.ok(signal);
+                // we check if we receive the signal with the birthday name
+                assert(signal.message.includes(config.applet.user.nameOfTheBirthdayPerson));
+                // check if the key is blinking
                 assert.equal(signal.points[0][0].effect, q.Effects.BLINK);
+                // check if the key is red
+                assert.equal(signal.points[0][0].color, '#F11512');
             }).catch((error) => {
                 assert.fail(error)
             });
         });
     });
 
     describe('#run()', () => {
-        it('the current date is far from the birthday date', async function () {
-            const simulatedDate = getConfigForDate(new Date(2019, 05, 21));
+        it('checks color of the selected keyboard key more than 1 month before the birthday date', async function () {
+            var date = new Date();
+            var dateTime = date.getTime();
+            // 60 secondes * 60 minutes * 24 heures
+            // On multiplie par 1000 car le time est exprimé en millisecondes
+            var endTime = dateTime + ((60*60*24) * 1000) * 75;
```

Comment:
Do not comment in French. 
Your method might work but it's not the best way to do it, use setDate instead.
Example aDate.setDate(today.getDate() + 3) // example for 3 days after today